Makefile

```diff
@@ -58,16 +58,13 @@ upgrade: $(COMMON_CONSTRAINTS_TXT)  ## update the requirements/*.txt files with
 ## Localization targets
 
 WORKING_DIR := lti_consumer
-EXTRACT_DIR := $(WORKING_DIR)/translations/en/LC_MESSAGES
+EXTRACT_DIR := $(WORKING_DIR)/conf/locale/en/LC_MESSAGES
 EXTRACTED_DJANGO := $(EXTRACT_DIR)/django-partial.po
-EXTRACTED_DJANGOJS := $(EXTRACT_DIR)/djangojs-partial.po
 EXTRACTED_TEXT := $(EXTRACT_DIR)/text.po
 
 extract_translations: ## extract strings to be translated, outputting .po files
 	cd $(WORKING_DIR) && i18n_tool extract
 	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
-	tail -n +20 $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT)
```

Comment:
The plugin has no text in JS files. so collecting is failing

Reply:
Are you referring to the `plugin` directory, or to this repository as a whole?

We do have text in the [xblock_lti_consumer.js](https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/static/js/xblock_lti_consumer.js) Javascript file. Although we've recently observed that those strings have not been translated in two years; we haven't investigated yet.

Does removing this line mean that those Javascript strings will not be translated?

Reply:
I've added a fix but then someone fixed it in a another PR https://github.com/openedx/xblock-lti-consumer/pull/353/files#diff-de32ac480ffc073d0eeb9f360281bd32ad2ad166d93821341685c945b414eb3c . So yes, we have JS translations